data/sql/updates/pending_db_world/rev_1583667245426891200.sql

```diff
@@ -517,8 +518,8 @@ UPDATE `creature_template` SET `mindmg` = 779, `maxdmg` = 1034, `DamageModifier`
 UPDATE `creature_template` SET `mindmg` = 448, `maxdmg` = 624, `DamageModifier` = 1 WHERE `entry` = 10438;
 -- Ramstein the Gorger
 UPDATE `creature_template` SET `mindmg` = 1190, `maxdmg` = 1578, `DamageModifier` = 1 WHERE `entry` = 10439;
--- Baron Rivendare (Not listed in Beastiary) Src: https://classicdb.ch/?npc=10440 
-UPDATE `creature_template` SET `mindmg` = 925, `maxdmg` = 1103 WHERE `entry` = 10440;
+-- Baron Rivendare (Not listed in Beastiary) Src: https://wotlk.evowow.com/?npc=10440 / https://db.rising-gods.de/?npc=10440 
+UPDATE `creature_template` SET `mindmg` = 351, `maxdmg` = 465, `DamageModifier` = 1 WHERE `entry` = 10440;
```

Comment:
na here it's probably a mistake, i preferred the previous change, looks way more logical, the end boss should hit almost 1k, maybe 700 but definitely not 465